docs/reference/scripting/using.asciidoc

```diff
@@ -241,9 +243,10 @@ templating language]. See <<search-template>> for more information and examples.
 
 All scripts are cached by default so that they only need to be recompiled
 when updates occur. By default, scripts do not have a time-based expiration, but
-you can change this behavior by using the `script.cache.expire` setting.
-You can configure the size of this cache by using the `script.cache.max_size` setting.
-By default, the cache size is `100`.
+you can configure the size of this cache by using the 
+`script.context.$CONTEXT.cache_expire` setting.
+By default, the cache size is `100` for all contexts except the `ingest` and the
```

Comment:
Consider making this a table. It might be easier to see the immediate differences, especially if we customize more contexts.

Reply:
tabled